llvm/test/CodeGen/SPARC/inlineasm-bad.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```

Comment:
Can UTC actually update error messages?...

Reply:
Ok. will drop this